### DISCUSSION TITLE
Packaging roadmap

### DISCUSSION THREAD

**Discussion by callumrollo** [c-proof/pyglider#33] [Ideas]
If we want to get pyglider more widely used, packaging it would be a good move.

What do we need to do to get to this point?

- Code cleanup (Issue #27)
- Remove legacy dependencies (Issue #20)
- Add tests for slocum 
- Make dependencies for pip (PyPI)
- Make conda-forge recipe
- Update README to reflect new capability & suggested workflow

The package doesn't need to be perfect, we can release it as a 0 tagged alpha so people will know it is all subject to change.

**Comment by jklymak** [comment]
Add to this, need docs.  I've started this, and should have a skeleton soon...

**Reply by jklymak** [reply]
Beginning of the docs done, and there is now a CI build for the docs, and the ability to view the build.   They are not deployed anywhere yet, but probably can be at some point.
